### ISSUE TITLE
"r is not a function"

### ISSUE THREAD

**Issue by DAOWAce** [DestinyItemManager/DIM#2420]
Getting this error message after the update today:

"Error loading Destiny info:
r is not a function
Reload to retry."

Needless to say, no matter how I reload, it still doesn't work.

Happens after downloading/unzipping.

Chrome x64

**Comment by bhollis** [comment]
What version of Chrome? Try upgrading?

**Comment by DAOWAce** [comment]
Chrome v51.

I'm unable to upgrade because of DirectWrite being forced on in v52+ and causing me severe eye strain when I attempt to read any text.

DIM worked absolutely fine before the 4.24 update, what changed?

**Comment by bhollis** [comment]
Can you use another browser? 51 is quite old and our compatibility target will age out old browsers. 

-Ben

**Comment by DAOWAce** [comment]
Changing browser is an extreme ordeal for me.

Firefox was my main browser.. but the whole Australis thing put an end to that _(and seemingly every update they'd change something nobody asked for, which caused an outcry and forced addons to be developed to change/bring features back)_. I switched from FF to PaleMoon when Australis hit.  PM didn't quite keep up with standards (like 60FPS Youtube videos, due to the DRM involved), and that's where Chrome came in.

I started using Chrome as a side browser for media, and besides, Firefox gradually turned into Chrome, but was full of half implemented features and ran worse, so I figured why not just use Chrome instead for modern stuff.

Eventually learned my FF/PM install ran **all** media content poorly, just skipped and stuttered and I could never figure out why. The only workaround was a completely new profile built from scratch.  6+ years of history in is in my profile (I'm an archivist; refuse to lose my history), guess the browser's just crippled under its own design.

After getting used to not being able to customize Chrome for crap, I settled with it as my main browser due to its speed.. then the DirectWrite thing eventually happened and forced me to prevent updating. One single change by two apparently ignorant developers totally screwed Chromium for tens of thousands of people.

Everything has been fine, mainly, but it seems slowly some things are just breaking with it.  I guess after a year and a half of changes to Chrome, it's been long enough for things to now not be compatible.

Sadly, most things are just forks of Firefox or Chrome.  Waterfox, Cyberfox.. Opera, Vivaldi.. it's all the same damn thing.  All Australis based Firefox forks are awful for me and all Chromium browser forks are awful (and Chromium is used in tons of things, like Steam, Discord, Battle.net, etc.).

That said, I learned that apparently [Cent brower](https://www.centbrowser.com/) is a fork of Chromium that **kept** the GDI code, which would mean it's an updated Chrome that's **actually usable**.  That info is a year out of date so I don't know if it still is, but if so, then this would solve my problem.. until I inevitably run into others.  Still need to set aside an entire day to attempt the switch.

Sorry to be a bother, but I hope keeping slightly older chrome support (if only for a bit longer) doesn't hamper your tool that much.
